spec.html

```diff
@@ -22443,13 +22462,17 @@ <h1>ResolveExport ( _exportName_, _resolveSet_ ) Concrete Method</h1>
                 1. Return ResolvedBinding Record { [[Module]]: _module_, [[BindingName]]: _e_.[[LocalName]] }.
             1. For each ExportEntry Record _e_ in _module_.[[IndirectExportEntries]], do
               1. If SameValue(_exportName_, _e_.[[ExportName]]) is *true*, then
-                1. Assert: _module_ imports a specific binding for this export.
                 1. Let _importedModule_ be ? HostResolveImportedModule(_module_, _e_.[[ModuleRequest]]).
-                1. Return _importedModule_.ResolveExport(_e_.[[ImportName]], _resolveSet_).
+                1. If _e_.[[ImportName]] is `"*"`, then
+                  1. NOTE: _module_ does not provide the direct binding for this export.
```

Comment:
Maybe this should be an assert as well for symmetry?

Reply:
Sounds good.